solidity/scripts/multiply-local-eth-btc-price.js

```diff
@@ -0,0 +1,29 @@
+const ETHBTCPriceFeedMock = artifacts.require("ETHBTCPriceFeedMock")
+
+async function run() {
+  try {
+    if (process.argv.length <= 4 || process.argv[4] === "help") {
+      console.log(
+        "run `truffle exec multiply-local-eth-btc-price.js <factor>` to update the current price by a factor of <factor>",
+      )
+    } else if (isNaN(parseFloat(process.argv[4]))) {
+      console.log("<factor> must be a number.")
+    } else {
+      const ethBtcPriceFeedMock = await ETHBTCPriceFeedMock.deployed()
+      const currentPrice = parseInt(await ethBtcPriceFeedMock.read())
+      // the arguments are [node, truffle, exec, <script name>, arguments...]
+      const factor = parseFloat(process.argv[4])
+      const newPrice = Math.round(currentPrice * factor)
```

Comment:
Strictly speaking this should probably use BN or BigInt for the math, otherwise we could overflow the precision of JavaScript's Number class. In practice I think this should work fine for testing though.